mappings/net/minecraft/client/gui/screen/Screen.mapping

```diff
@@ -67,7 +67,8 @@ CLASS net/minecraft/class_437 net/minecraft/client/gui/screen/Screen
 	METHOD method_25428 confirmLink (Z)V
 		ARG 1 open
 	METHOD method_25429 addChild (Lnet/minecraft/class_364;)Lnet/minecraft/class_364;
-	METHOD method_25430 handleComponentClicked (Lnet/minecraft/class_2561;)Z
+	METHOD method_25430 handlTextClicked (Lnet/minecraft/class_2561;)Z
```

Comment:
Typo.
I personally think this should be either `handleTextClick` or `onTextClicked`.